### DISCUSSION TITLE
Blue line in the drop down menu in mobile version

### DISCUSSION THREAD

**Discussion by jaymnetwork** [quarto-dev/quarto-cli#8911] [Q&A] [answered]
### Description

Hello, I'm trying to publish a website using quarto, and it works perfectly fine when I see it through my desktop (both desktop and mobile version). But when I see the website with my iphone, and click the drop down menus, I see this blue lines.

![IMG_7743](https://github.com/quarto-dev/quarto-cli/assets/158008375/4778b048-5164-4f26-b2fe-fb02b31f83b8)

This is what I have at the very bottom of the .yml file
`
format:
  html:
    css: styles.css
    toc: true
`

And I also have tried the inserting following codes in my css file but didn't work either.

`
.header-menu-sidebar .nav-ul a {
    outline: none !important;
}
`

`
.et_mobile_menu {
  border-top: 2px solid
  ##ff5733!important;
}
`

Do you have any ideas why these blue lines are showing in the mobile version?

**Comment by mcanouil** [comment]
Could you share a small **self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.


See <https://quarto.org/docs/output-formats/html-themes.html> and use your favourite browser to inspect HTML and target the proper elements.


---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by jaymnetwork** [reply, marked as answer]
Hello, I was able to solve it by inserting `outline: none;` in the css file! but thank you for trying to help me out!

**Reply by mcanouil** [reply]
Glad you solved it!
